components/cdc/src/delegate.rs

```diff
@@ -4,10 +4,22 @@ use std::mem;
 use std::sync::atomic::{AtomicBool, AtomicUsize, Ordering};
 use std::sync::Arc;
 
-use futures::sync::mpsc::*;
+#[cfg(not(feature = "prost-codec"))]
 use kvproto::cdcpb::*;
+#[cfg(feature = "prost-codec")]
+use kvproto::cdcpb::{
+    event::{
+        row::OpType as EventRowOpType, Entries as EventEntries, Error as EventError,
```

Comment:
Why have the difference? Can it be solved by protobuf-build?

Reply:
We could write these alias in kvproto, but I want to merge this PR quickly as it blocks other CDC PRs.